python_modules/libraries/dagster-aws/setup.py

```diff
@@ -36,7 +36,7 @@ def get_version() -> str:
             "boto3",
             f"dagster{pin}",
             "packaging",
-            "psycopg2-binary<2.9",
+            "psycopg2-binary",
```

Comment:
Are we at all worried that changing pins will cause release hiccups? I think we've decided that relaxing pins should be safe but adding pins has caused us build issues in the past - so I think we're fine?

Reply:
the pins we had problems with in the past were in the automation setup.py too - I don't **think** these library deps should cause any issues